tokenizers/src/models/unigram/model.rs

```diff
@@ -573,4 +605,191 @@ mod tests {
             assert_eq!(model.encode("abqrcd").unwrap(), vec!["ab", "q", "r", "cd"]);
         }
     }
+
+    #[test]
+    fn test_unigram_bytefallback() {
+        // In [97]: processor.encode_as_pieces("⅐⅛⅑ ")
+        // Out[97]: ['▁', '<0xE2>', '<0x85>', '<0x90>', '⅛', '<0xE2>', '<0x85>', '<0x91>', '▁']
+        let sentencepieces = vec![
+            ("<unk>".to_string(), 0.0),
+            ("<0xE2>".to_string(), -0.01),
+            ("<0x85>".to_string(), -0.02),
+            ("<0xE2>".to_string(), -0.03),
+            ("<0x90>".to_string(), -0.04),
+            ("<0x91>".to_string(), -0.05),
+            ("<0x9B>".to_string(), -0.06),
+            ("Hello".to_string(), -0.1),
+            ("how".to_string(), -0.2),
+            ("are".to_string(), -0.3),
+            (" ".to_string(), -0.4),
+        ];
+        let mut unigram = Unigram::from(sentencepieces, Some(0), true).unwrap();
+        let tokens = unigram.tokenize("Hello⅐how are⅑⅑").unwrap();
```

Comment:
Can we make these examples smaller and more readable ?

We only need 1 utf-8 multibyte token `"é"` could work (we can keep your tokens it's fine).
`"Hello⅐"` should be enough to prove the point.

Then instead of using a new utf-8 that we don't know how it's encoded, just recreate the vocab with only 1 missing byte.

And the `fuse_unk` shouldn't need to be tested. It only has impact on the offsets, which is quite orthogonal with what we're testing here, no ?

Reply:
No `fuse_unk` is tested to make sure that if there more than 1 byte, all of them are returned (we disccused about this offline)

Reply:
Plus we want to make sure that when 1 token in the unk sequence does not have bytefallback, then the entire string is returned as unk, instead of falling back for the known one!

Reply:
Reduce the examples 😉